wgpu-core/Cargo.toml

```diff
@@ -30,7 +30,7 @@ log = "0.4"
 hal = { package = "gfx-hal", version = "0.5.2" }
 gfx-backend-empty = "0.5"
 gfx-descriptor = "0.1"
-gfx-memory = "0.1"
+gfx-memory = { version = "0.1", git = "https://github.com/gfx-rs/gfx-extras", rev = "438353c3f75368c12024ad2fc03cbeb15f351fd9" }
```

Comment:
let's put it out separately now, like we do with `naga`
also, version isn't needed if we are pointing to github